docs/janssen-server/fido/logs.md

```
@@ -63,17 +65,27 @@ Response:
+
+### Update logging level
+
+1. Create a JSON file say `/tmp/config_values.json` by editing the JSON from Point 1
+
+      - edit `loggingLevel` to `TRACE` or `DEBUG` or `INFO`
+      
+      - edit `loggingLayout` to `text` or `json`
+
+2. Use the following command to update the logging level
+
+      ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)